conf/fink_alert_simulator.conf

```diff
@@ -19,8 +19,9 @@
 KAFKA_PORT_SIM=29092
 KAFKA_IPPORT_SIM="localhost:${KAFKA_PORT_SIM}"
 
-# Cluster mode - you would a Kafka cluster installed with write mode.
-KAFKA_IPPORT=""
+# Cluster mode - require a Kafka cluster installed with write mode.
+KAFKA_PORT=31734
+KAFKA_IPPORT="192.168.49.2:${KAFKA_PORT}"
```

Comment:
What is this IP address?

Reply:
:+1: It is an example IP address, I edit the file to make it more understandable.